src/avgScrabbleScoreFromFile.js

```diff
@@ -11,12 +11,16 @@ let avgScrabbleScore = require('./avgScrabbleScore');
  * sample-words.txt.
  */
 function avgScrabbleScoreFromFile(fileName) {
-  // This is your job. :)
+  let arrayOfWords = fs.readFileSync(`../src/${fileName}`, 'utf-8').split('\n');
```

Comment:
Careful; there's a slight bug here. Create a simple test file and make sure it's returning what you expect, e.g.,

```text
aaa
aaa
aaa
```

Also, I wouldn't hard code `../src/${fileName}` here. The caller would have to know this detail in order to know what to pass to the function.

Instead, have the caller call something like `avgScrabbleScoreFromFile('../src/sample-words.txt')` if that's what they want.